client/extensions/wp-super-cache/accepted-filenames.jsx

```diff
@@ -48,6 +49,28 @@ const AcceptedFilenames = ( {
 			</SectionHeader>
 			<Card>
 				<form>
+					<FormLabel>
+						{ translate( 'Do not cache these page types.' ) }
+					</FormLabel>
+
+					<FormSettingExplanation className="wp-super-cache__condition-settings-explaination">
+						{ translate(
+							' See the {{a}}Conditional Tags{{/a}} ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Do not cache these page types. See the {{a}}Conditional Tags{{/a}} documentation for a complete discussion on each type.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).